packages/nimble-components/src/rich-text/specs/README.md

```diff
@@ -473,6 +983,47 @@ _Keyboard navigation with toolbar buttons focused_
 
 _Note_: All other keyboard interaction determined by the slotted element will not be defined in this document.
 
+_Keyboard interactions for `@mention`_
+
+| Key                                      | Behavior                                                                                                                            |
+| ---------------------------------------- | ----------------------------------------------------------------------------------------------------------------------------------- |
+| `@`                                      | To open the list of options with the users list                                                                                     |
```

Comment:
I noticed in the prototype that the users list doesn't appear if the `@` is preceeded by a character other than space. This seems good so that it doesn't appear when typing an email address. It'd be good to mention that behavior explicitly here.

Reply:
Added this behavior too